routecontroller/integration/integration_test.go

```diff
@@ -1053,6 +1054,69 @@ var _ = Describe("Integration", func() {
 			))
 		})
 	})
+
+	FDescribe("kubectl", func() {
+		type routeView struct {
+			name string
+			url  string
+			age  string
+		}
+
+		When("viewing routes with -owide view mode", func() {
+			BeforeEach(func() {
+				yamlToApply = filepath.Join("fixtures", "multiple-routes-with-different-fqdn.yaml")
+			})
+
+			It("outputs the associated name and URL", func() {
+				Eventually(func() ([]routeView, error) {
+					output, err := kubectlWithConfig(kubeConfigPath, nil, "-n", namespace, "get", "routes", "-o", "wide")
+					if err != nil {
+						return nil, err
+					}
+
+					// the first line is a header with column names
+					lines := strings.Split(string(output), "\n")[1:]
+
+					routes := make([]routeView, len(lines))
+
+					const (
+						nameColumn = 0
+						urlColumn  = 1
+						ageColumn  = 2
+					)
+
+					spaceRe := regexp.MustCompile(`\s+`)
+					for _, line := range lines {
+						if line == "" {
+							continue
+						}
+
+						columns := spaceRe.Split(line, -1)
+						Expect(columns).Should(HaveLen(3))
+
+						routes = append(routes, routeView{
+							name: columns[nameColumn],
+							url:  columns[urlColumn],
+							age:  columns[ageColumn],
+						})
+					}
+
+					return routes, nil
+				}).Should(ConsistOf(
+					routeView{
+						name: "cc-route-guid-1",
+						url:  "hostname-1.apps.example.com/some/path",
+						age:  "0s",
```

Comment:
Not sure this should be tested

Reply:
I decided not to test the exact value rather that the field is not empty.
The reason I have to test this is that if you add additional print columns and do not include age column (which is included by default) it won't be shown.